### PR TITLE
sunset DISPLAY_BASE_TOKEN and COLLATERAL_BASE_TOKEN

### PR DESCRIPTION
**Pull request by ShawTim**

### CODE REVIEW

File: src/components/OpenPosition/OpenPosition.tsx [hunk 1]
```diff
@@ -36,6 +37,9 @@ import { PositionAfter, PositionAction } from '../Position';
 
 import './OpenPosition.scss';
 
+// TODO: SDK will provide this
+const TOKENS_WITH_PERMIT = new Set(['wstETH', 'WETH', 'R']);
```
Comment (lazarmitic) [inline src/components/OpenPosition/OpenPosition.tsx:41]: WETH does not have permit I think.

File: src/components/Position/PositionAfter.tsx [hunk 2]
```diff
@@ -33,13 +35,13 @@ export const PositionAfter: FC<PositionAfterProps> = ({
 }) => {
   const baseTokenAmountFormatted = useMemo(
     () =>
-      DecimalFormat.format(displayCollateralToken ?? Decimal.ZERO, {
+      DecimalFormat.format(displayCollateralTokenAmount ?? Decimal.ZERO, {
         style: 'currency',
-        currency: DISPLAY_BASE_TOKEN,
+        currency: TOKEN_TO_DISPLAY_BASE_TOKEN_MAP[displayCollateralToken],
```
Comment (lazarmitic) [inline src/components/Position/PositionAfter.tsx:40]: Prop `displayCollateralToken` is already a display token, no need to use `TOKEN_TO_DISPLAY_BASE_TOKEN_MAP` here I think.

File: src/components/Position/PositionAfter.tsx [hunk 3]
```diff
@@ -77,7 +79,7 @@ export const PositionAfter: FC<PositionAfterProps> = ({
         <ul className="raft__position-after__data__position__data">
           <li className="raft__position-after__data__position__data__deposit">
             <TooltipWrapper tooltipContent={<TokenAddressTooltip />} placement="top">
-              <TokenLogo type={`token-${DISPLAY_BASE_TOKEN}`} size={20} />
+              <TokenLogo type={`token-${TOKEN_TO_DISPLAY_BASE_TOKEN_MAP[displayCollateralToken]}`} size={20} />
```
Comment (lazarmitic) [inline src/components/Position/PositionAfter.tsx:82]: Same here.

Comment (ShawTim) [reply src/components/Position/PositionAfter.tsx]: fixed

File: src/components/Position/PositionAfter.tsx (hunk 2 above)
Comment (ShawTim) [reply src/components/Position/PositionAfter.tsx]: fixed

File: src/components/OpenPosition/OpenPosition.tsx (hunk 1 above)
Comment (ShawTim) [reply src/components/OpenPosition/OpenPosition.tsx]: removed in this case

**Pull request merged**